test_pytest.py

```diff
@@ -34,6 +34,34 @@ def test_experience():
     assert response.json[item_id] == example_experience
 
 
+
```

Comment:
Why is this all commented out? You can remove this test since the post endpoint is not there